pokete.py

```diff
@@ -1216,8 +1262,17 @@ def __call__(self):
 
 class About:
     """The about text, that can be triggered in the menu"""
+
     def __init__(self):
-        self.box = InfoBox(liner(f"Pokete v{VERSION} -- {CODENAME}\n by lxgr-linux <lxgr@protonmail.com>\n \n This software is licensed under the GPL3, you should have gotten a copy of the GPL3 license alongside this software.\n Feel free to contribute what ever you want to this game, new Pokete contributions are especially welcome.\n For this see the comments in the definations area.\n You can contribute here: https://github.com/lxgr-linux/pokete", 60, pre=""), map=movemap)
+        self.box = InfoBox(liner(f"""
```

Comment:
I guess there was a cause I didn't do that in the first place.

Reply:
Wow. So se bug?

Reply:
Fixed in 0e3baa2ea0987955c57bd19a4e5fd05a1abaff4f

Reply:
Just use two spaces in a new line and everything should be fine.